.ci/docker_config.json

```
@@ -0,0 +1,36 @@
+{
+    "debug": false,
```

```suggestion
    "debug": true,
```
I mean, why not? :) It will (hopefully) make debugging in the CI easier as well.